src/dmd/dinterpret.d

```diff
@@ -698,6 +698,17 @@ private Expression interpretFunction(UnionExp* pue, FuncDeclaration fd, InterSta
     return e;
 }
 
+void incUsageCtfe(InterState* istate, const ref Loc loc)
+{
+    if (global.params.cov && istate)
+    {
+        auto line = loc.linnum;
+        auto mod = istate.fd.getModule();
+
+        ++mod.ctfe_cov[line];
```

Comment:
This could just be a one liner function.

Reply:
could it? Where do I get the currently executed module?

Reply:
Ahhh you mean `++istate.fd.getModule().ctfe_cov[loc.linnum];`
Yes I _could_ write that. I find it to be less readable however.